src/integrationTest/java/net/sf/jabref/gui/EntryTableTest.java

```diff
@@ -0,0 +1,91 @@
+package net.sf.jabref.gui;
+
+import java.awt.event.KeyEvent;
+import java.io.File;
+import java.util.regex.Pattern;
+
+import net.sf.jabref.JabRefMain;
+
+import org.assertj.swing.fixture.FrameFixture;
+import org.assertj.swing.fixture.JFileChooserFixture;
+import org.assertj.swing.fixture.JTableCellFixture;
+import org.assertj.swing.fixture.JTableFixture;
+import org.assertj.swing.junit.testcase.AssertJSwingJUnitTestCase;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.assertj.swing.finder.WindowFinder.findFrame;
+import static org.assertj.swing.launcher.ApplicationLauncher.application;
+
+
+/**
+ * Specific Use-Case:
+ * I import a database. Then I doubleclick on the first entry in the table to open the entry editor.
+ * Then I click on the first entry again, and scroll through all of the lists entries, without having to click
+ * on the table again.
+ */
+public class EntryTableTest extends AssertJSwingJUnitTestCase {
+
+    private final static int SCROLL_ACTION_EXECUTION = 5;
+    private final static String TEST_FILE_NAME = "testjabref.bib";
+
+    private int down = KeyEvent.VK_DOWN;
+
+    @Override
+    protected void onSetUp() {
+        AWTExceptionHandler awtExceptionHandler = new AWTExceptionHandler();
+        awtExceptionHandler.installExceptionDetectionInEDT();
+        application(JabRefMain.class).start();
+
+
+        robot().settings().timeoutToFindSubMenu(1_000);
+        robot().settings().delayBetweenEvents(50);
+    }
+
+    @Test
+    public void scrollThroughEntryList() throws InterruptedException {
+        FrameFixture mainFrame = findFrame(JabRefFrame.class).withTimeout(10_000).using(robot());
+        robot().waitForIdle();
+
+        mainFrame.menuItemWithPath("File", "Import into new database").click();
+        JFileChooserFixture openFileChooser = mainFrame.fileChooser();
+
+        //reduce the delay to speed up loading the test resource
+        robot().settings().delayBetweenEvents(10);
+        openFileChooser.fileNameTextBox().enterText(getTestFilePath(TEST_FILE_NAME));
```

Comment:
If I see right, you just want the file path as a string. So I don't think you need getTestFilePath method if you're using java.nio.path. 
See [here] (http://www.java2s.com/Tutorials/Java/java.nio.file/Path/Java_Path_getFileName_.htm) for an example.